src/components/Pensjonsavtaler/utils.tsx

```diff
@@ -19,3 +19,52 @@ export const groupPensjonsavtalerByType = (
   }
   return record
 }
+
+export const finnAllePensjonsavtalerVedUttak = <
```

Comment:
Fjern for ordens skyld `export` siden den ikke brukes andre steder
```suggestion
const finnAllePensjonsavtalerVedUttak = <
```